### ISSUE TITLE
Error: The local development server was not able to start.

### ISSUE THREAD

**Issue by romuloroc** [forcedotcom/lwc-dev-server-feedback#137]
Issue Type: <b>Feature Request</b>

Error: The local development server was not able to start.

Extension version: 54.6.1
VS Code version: Code 1.65.2 (c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1, 2022-03-10T14:33:55.248Z)
OS version: Windows_NT x64 10.0.19043
Restricted Mode: No

**Comment by randi274** [comment]
Hi @romuloroc I've moved your issue over to our [LWC Dev Server Feedback](https://github.com/forcedotcom/lwc-dev-server-feedback) repository, where we track issues related to that product separately from our extensions. Are you still facing this issue?

**Comment by romuloroc** [comment]
> Oi@romulorocMovi seu problema para nosso repositório de [comentários do LWC Dev Server](https://github.com/forcedotcom/lwc-dev-server-feedback) , onde rastreamos problemas relacionados a esse produto separadamente de nossas extensões. Você ainda está enfrentando esse problema?

Yes, now npm asks for a validation in "405 Method Not Allowed - GET https://registry.npmjs.org/NPM/windows-build-tools". I can't install the server.

**Comment by SaurabhGyanSys** [comment]
Hi, Can anyone help with the installation of the Local Dev Server in LWC? I tried but I got the error. 
I tried to with this command **sfdx plugins:install @salesforce/lwc-dev-server** and got the error 
![image](https://user-images.githubusercontent.com/112639498/232464750-a7f991fb-095f-43a1-a566-f4d9c9813bd7.png)
and after I tried to resolve this issue and install the node -gyp with this command **npm install -g node-gyp**  nothing happened.
Help ,e out

**Comment by akashsnair05** [comment]
First Authorize the org then connect it local server!
